src/validators/post.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+// eslint-disable-next-line @typescript-eslint/no-var-requires,@typescript-eslint/no-unsafe-assignment
+
+import { schemas } from '@dvsa/cvs-type-definitions/lib/schemas';
+
+export const identifyObjectType = (obj: any, method: string) => identifySchema(obj.techRecord_vehicleType, obj.techRecord_recordCompleteness, method);
+export const identifySchema = (vehicleType: string, recordCompleteness: string, method: string) => schemas
+  .filter((x: string | string[]) => (x.includes(vehicleType) && x.includes(recordCompleteness)))
+  .filter(((x: string | string[]) => x.includes(method)))
+  .map((x) => x);
```

Comment:
```suggestion
import { schemas } from '@dvsa/cvs-type-definitions/lib/schemas';

export const identifyObjectType = (obj: any, method: string) => identifySchema(obj.techRecord_vehicleType, obj.techRecord_recordCompleteness, method);
export const identifySchema = (vehicleType: string, recordCompleteness: string, method: string) => schemas
  .filter(x => x.includes(vehicleType) && x.includes(recordCompleteness) && x.includes(method))
```
would be good have types/enums for the input of this function